cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -1706,4 +1797,33 @@ x"
             Err(e) => panic!("Incorrect error returned {}", e),
         }
     }
+    #[test]
+    fn test_parse_param() {
+        use super::HashSet;
```

Comment:
I think we should add these two `use`s to the beginning of the `test` module (and prefer `std::collections::HashSet` over `super::HashSet`).